jobs.groovy

```diff
@@ -609,6 +614,11 @@ def define_job(dashProject, dashBranchName, branchName, job_type, job_name,
         wrappers build_wrappers(job_values)
         triggers build_triggers(job_type, job_values.at, branchName)
         scm build_scm(git_url, branchName, isReleaseBuild)
+        steps {
+          shell(updateGitHubStatus(
+                'pending', 'Build started',
+                branchName, dashProject, dashBranchName, job_name))
+        }
```

Comment:
This could go in `build_steps`, but we don't have all the variables we need there. I could change that, but I'm not sure the value of that.